src/main/resources/messages.properties

```diff
@@ -474,5 +474,10 @@ validation.date.invalid.directors_approval.date=The date cannot be on or before
 validation.characters.invalid.off_balance_sheet_arrangements.details=Enter valid characters
 validation.length.invalidInputLength.off_balance_sheet_arrangements.details=You must not exceed {1} characters
 
+# Loans To Directors
 
+loanToAdd.directorName = Director name:
+validation.element.missing.loan.director_name = Enter the director's name
 
+validation.characters.invalid.loan.director_name = Enter valid characters
+validation.length.invalidInputLength.loan.director_name=You must not exceed {1} characters
```

Comment:
As above comment

Reply:
These are used in the ValidationContext along with the @Validmodel annotation on the model being validated.